en/telemetry/esp32_wifi_module.md

```diff
@@ -0,0 +1,149 @@
+* [Adafruit AirLift – ESP32 WiFi Co-Processor Breakout Board](https://www.adafruit.com/product/4201) (requires FTDI adapter for flashing firmware)
+* [Adafruit HUZZAH32](https://www.adafruit.com/product/4172) (requires FTDI adapter for flashing firmware)
+Boards with an IPEX port for an external antenna often also offer an onboard antenna that is activated by default. You 
+  may need to resolder a resistor to activate the external antenna port.
+## Downloading and Flashing the Firmware
+[Download the firmware from the GitHub repository](https://github.com/DroneBridge/ESP32/releases) and
+[follow the flashing instructions there](https://github.com/DroneBridge/ESP32#installationflashing-using-precompiled-binaries). 
+They are always up to date. 
+:::tip
+[Follow the flashing instructions inside the GitHub Repository.](https://github.com/DroneBridge/ESP32#installationflashing-using-precompiled-binaries) 
+The exact parameters may differ from release to release of DroneBridge for ESP32.
+:::
+For convenience reasons some short instructions are given here:
+* [Download the pre-compiled firmware binaries](https://github.com/DroneBridge/ESP32/releases)
+* Connect your DEVKit to your computer via USB/Serial bridge (most DevKits already offer a USB port for flashing and debugging)
+* Erase the flash and flash the DroneBridge for ESP32 firmware onto your ESP32
+  * Using [Espressif Flash Download Tool](https://www.espressif.com/en/support/download/other-tools) (Windows only)  
+  * Using esp-idf/esptool (all platforms)  
+* Power Cycle the ESP32
+* [Connect to the "DroneBridge for ESP32" WiFi network and configure the firmware for your application](#configuring-dronebridge-for-esp32)
+
+## Configuring DroneBridge for ESP32
+
+### Default Configuration
+* SSID: `DroneBridge for ESP32`
+* Password: `dronebridge`
+* Transparent/MAVLink
+* UART baud rate `115200`
+* UART TX pin `17`
+* UART RX pin `16`
+* Gateway IP: `192.168.2.1`
+
+### Custom Settings & Webinterface
+
+You can change the default configuration via the Webinterface.  
+Connect to the ESP32 via WiFi and enter `dronebridge.local`, `http://dronebridge.local` or `192.168.2.1` in the address 
+bar of your browser.
+
+![DroneBridge for ESP32 Webinterface](https://raw.githubusercontent.com/DroneBridge/ESP32/master/wiki/dbesp32_webinterface.png)
+
+:::tip
+Some settings require you to reboot the ESP32 to take effect.
+:::
+
+## Wiring
+Wiring is very simple and mostly the same for all devices connected to the TELEM1/2 ports of a Pixhawk. That is why this 
+guide does not go into detail here.
+
+* Connect UART of ESP32 to a UART of your flight controller (e.g. TELEM 1 or TELEM 2 port). Make sure the voltage levels 
+  match! Most ESP32 DevKits can only take 3.3V!
+  * TX to RX
+  * RX to TX
+  * GND to GND
+  * Stable 3.3V or 5V power supply to the ESP32 (depending on the available inputs of your DevKit)
+* Set the flight controller port to the desired protocol.
```

Comment:
@seeul8er Do you mean enable mavlink on the port as per ... http://docs.px4.io/master/en/peripherals/mavlink_peripherals.html ?